src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/PbfToRawAtlasCommand.java

```diff
@@ -0,0 +1,177 @@
+package org.openstreetmap.atlas.utilities.command.subcommands;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openstreetmap.atlas.geography.atlas.packed.PackedAtlas;
+import org.openstreetmap.atlas.geography.atlas.packed.PackedAtlas.AtlasSerializationFormat;
+import org.openstreetmap.atlas.geography.atlas.raw.creation.RawAtlasGenerator;
+import org.openstreetmap.atlas.geography.sharding.CountryShard;
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.streaming.resource.FileSuffix;
+import org.openstreetmap.atlas.utilities.command.AtlasShellToolsException;
+import org.openstreetmap.atlas.utilities.command.abstractcommand.CommandOutputDelegate;
+import org.openstreetmap.atlas.utilities.command.abstractcommand.OptionAndArgumentDelegate;
+import org.openstreetmap.atlas.utilities.command.parsing.ArgumentArity;
+import org.openstreetmap.atlas.utilities.command.parsing.ArgumentOptionality;
+import org.openstreetmap.atlas.utilities.command.parsing.OptionOptionality;
+import org.openstreetmap.atlas.utilities.command.subcommands.templates.MultipleOutputCommand;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author samgass
+ */
+public class PbfToRawAtlasCommand extends MultipleOutputCommand
+{
+    private static final Logger logger = LoggerFactory.getLogger(PbfToRawAtlasCommand.class);
+
+    // The filetype for PBFs-- will strip this from the filename to make the Atlas filename
+    private static final String PBF_FILETYPE = ".pbf";
```

Comment:
Check out the `FileSuffix` enum for this. `FileSuffix.PBF.toString()`

Reply:
oh good this seemed really hacky but I didn't know about FileSuffix

Reply:
It's a @MikeGost classic. I evangelize it at every opportunity.